generators/languages/templates/src/main/webapp/i18n/en/_activate.json

```diff
@@ -3,9 +3,9 @@
         "title": "Activation",
         "messages": {
             <%_ if (clientFramework === 'angular1') { _%>
-            "success": "<strong>Your user has been activated.</strong> Please <a class=\"alert-link\" href=\"\" ng-click=\"vm.login()\">sign in</a>.",
+            "success": "<strong>Your user account has been activated.</strong> Please <a class=\"alert-link\" href=\"\" ng-click=\"vm.login()\">sign in</a>.",
             <%_ } else { _%>
-            "success": "<strong>Your user has been activated.</strong> Please ",
+            "success": "<strong>Your user account has been activated.</strong> Please sign in.",
```

Comment:
OK, I only added this after I only saw "Your user account has been activated. Please" with no further message or link on the UI.  I'll look into what may be broken on my setup then.